test/e2e/framework/framework.go

```diff
@@ -317,17 +320,31 @@ func (f *Framework) AfterEach() {
 	}
 
 	outputTypes := strings.Split(TestContext.OutputPrintType, ",")
+	now := time.Now()
 	for _, printType := range outputTypes {
 		switch printType {
 		case "hr":
 			for i := range summaries {
-				Logf(summaries[i].PrintHumanReadable())
+				if TestContext.ReportDir == "" {
+					Logf(summaries[i].PrintHumanReadable())
+				} else {
+					filePath := path.Join(TestContext.ReportDir, summaries[i].SummaryKind()+now.Format(time.RFC3339)+".txt")
```

Comment:
Good point. What we want is the "KubeDescribe" part of it. Anyway. TODO is fine.